public/static/pathways/upenn_her2_early_stage_pathway.json

```diff
@@ -0,0 +1,589 @@
+            "condition": {
+              "description": "T <= 2",
+              "cql": "[Observation: \"Primary tumor.clinical [Class] Cancer code\"] TSmall where ToConcept(TSmall.value as FHIR.CodeableConcept) ~ \"T0 category (finding)\" or ToConcept(TSmall.value as FHIR.CodeableConcept) ~ \"T1 category (finding)\" or ToConcept(TSmall.value as FHIR.CodeableConcept) ~ \"T2 category (finding)\" return Tuple{ resourceType: 'Observation', id: TSmall.id.value , status: TSmall.status.value} "
+            }
+          }
+        ]
+      },
+      "NodeStatus": {
+        "label": "Node Status",
+        "transitions": [
+          {
+            "transition": "ChemotherapyTCHP",
+            "condition": {
+              "description": "N+",
+              "cql": "[Observation: \"Regional lymph nodes.clinical [Class] Cancer code\"] NLarge let NLargeConcept: ToConcept(NLarge.value as FHIR.CodeableConcept) where NLargeConcept ~ \"N1 category (finding)\" or NLargeConcept ~ \"N2 category (finding)\" or NLargeConcept ~ \"N3 category (finding)\" return Tuple{ resourceType: 'Observation', id: NLarge.id.value , status: NLarge.status.value} "
+            }
+          },
+          {
+            "transition": "ChemotherapyACTHP",
+            "condition": {
+              "description": "N+",
+              "cql": "[Observation: \"Regional lymph nodes.clinical [Class] Cancer code\"] NLarge let NLargeConcept: ToConcept(NLarge.value as FHIR.CodeableConcept) where NLargeConcept ~ \"N1 category (finding)\" or NLargeConcept ~ \"N2 category (finding)\" or NLargeConcept ~ \"N3 category (finding)\" return Tuple{ resourceType: 'Observation', id: NLarge.id.value , status: NLarge.status.value} "
+            }
+          },
+          {
+            "transition": "ChemotherapyddACTHP",
+            "condition": {
+              "description": "N+",
+              "cql": "[Observation: \"Regional lymph nodes.clinical [Class] Cancer code\"] NLarge let NLargeConcept: ToConcept(NLarge.value as FHIR.CodeableConcept) where NLargeConcept ~ \"N1 category (finding)\" or NLargeConcept ~ \"N2 category (finding)\" or NLargeConcept ~ \"N3 category (finding)\" return Tuple{ resourceType: 'Observation', id: NLarge.id.value , status: NLarge.status.value} "
+            }
+          },
+          {
+            "transition": "Surgery1",
+            "condition": {
+              "description": "N0",
+              "cql": "[Observation: \"Regional lymph nodes.clinical [Class] Cancer code\"] N0 where ToConcept(N0.value as FHIR.CodeableConcept) ~ \"N0 category (finding)\" return Tuple{ resourceType: 'Observation', id: N0.id.value , status: N0.status.value} "
+            }
+          }
+        ]
+      },
+      "Surgery1": {
+        "label": "Surgery",
+        "action": [
+          {
+            "type": "create",
+            "description": "Cancer Related Surgery Procedure",
+            "resource": {
+              "resourceType": "Procedure",
```

Comment:
I'm not sure about using `Procedure` resources in the action here. (Understood that we have `Procedure`s in the older pathways)
On one hand I think we should be using only "Request" type resources in the `action` field, for example in this case a `ServiceRequest`(R4)/`ProcedureRequest`(STU3/DSTU2). When the oncologist clicks the accept button here, they aren't saying the surgery is happening right now or already done, they are requesting the action to happen.  
But on the other hand that could potentially also be represented as a `Procedure` with `status`: `preparation`.
Let's discuss after standup today.

Also yes I'm aware using `ServiceRequest` here makes the CQL more difficult as we'd then want to check for either the proecedure request or performed procedure. Maybe there's something we could extract as a helper function to make that easier.

Reply:
Hmm that is a good point. Could we not do a Request type for the resource and then keep the CQL to check for the actual procedure to be complete? I don't think there is a link between the CQL and the action resource type

Reply:
We want the CQL to be able to recognize whether the node is not started at all, in-progress, or completed, so I think the CQL has to look for both, or if just one, then the request. If the user clicks on the accept button and the app writes back a request, we want to recognize the next time that that they clicked that button and the node is at least in-progress.

Reply:
From a quick meeting with Dylan and Reece this branch is going to modify the CQL of the action states to return either the service request or the actual procedure if it exists. From there a future task can mark nodes progress based on the status of the returned documentation resource.